### PR TITLE
Override identification of gzipped WARCs

### LINKED ISSUE #343: Identify WARC files as WARCs, not zips
The technical metadata service is identifying WARC files as application/gzip. This is not wrong - in that the files are gzipped - but it would be better from a repository management perspective if they were identified as WARCs. 

I'm not sure what the path would be to achieve that, given this is coming from the output of the Siegfried file identification tool. In the structural metadata, these items are being identified as application/warc, but I think that's because they're being processed by a special "metadataExtractor.jar" file in the wasCrawlPreassemblyWF.

### PR DESCRIPTION
**Pull request by mjgiarlo**
## Why was this change made? 🤔

Fixes #343
    
This commit adds logic to the `FileIdentifierService` allowing it to detect when it has been asked to identify a gzipped WARC file and, when it has, to return the file format information of a non-gzipped WARC file. Requested by product owner to provide more useful information about web archives.

Also includes:

* Point app and worker docker-compose services at docker Redis URL: without this commit, `docker-compose up workers` would not successfully bring up the workers.

## How was this change tested? 🤨

CI, and run locally against the new fixture WARC.

### CODE REVIEW

**Review by ndushay — APPROVED** [review]
LGTM

File: app/services/file_identifier_service.rb [hunk 1]
```diff
@@ -14,7 +14,12 @@ def identify(filepath:)
     output, err, status = Open3.capture3('sf', '-json', filepath)
     raise Error, "Identifying #{filepath} returned #{status.exitstatus}: #{err}\n#{output}" unless status.success?
 
-    extract_file_types(output, filepath)
+    pronom_id, mimetype = extract_file_types(output, filepath)
+    return [pronom_id, mimetype] unless pronom_id == 'x-fmt/266' &&
```
Comment (jcoyne) [inline app/services/file_identifier_service.rb:18]: Doesn't the first clause (`x-fmt/266`) imply the second (`application/gzip`)?  Do we need both clauses?

File: docker-compose.yml [hunk 2]
```diff
@@ -13,6 +13,7 @@ services:
       DATABASE_PORT: 5432
       RAILS_LOG_TO_STDOUT: "true"
       REDIS_URL: redis://redis:6379/
+      SETTINGS__REDIS_URL: redis://redis:6379/
```
Comment (jcoyne) [inline docker-compose.yml:16]: Why is this required in this pr?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (jcoyne) [reply docker-compose.yml:16]: Ah, I see the separate commit message.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**